src/components/Nav/MegaMenu/index.tsx

```diff
@@ -0,0 +1,693 @@
+import React, { useContext, useRef, useState } from 'react';
+import ArrowForLink from 'components/UI/ArrowForLink';
+import Link from 'components/UI/Link';
+import { getDetailOrSearchURLForQuery } from 'utils/accessions';
+import { createParamFromURL } from 'hooks/queryParamState/QueryParamStore/queryParamReducer';
+import useQueryParamsStore from 'hooks/queryParamState/QueryParamStore/useQueryParamsStore';
+import { useNavigate } from 'react-router-dom';
+import UserContext from 'pages/Login/UserContext';
+
+const MegaMenu: React.FC = () => {
+  const { isAuthenticated } = useContext(UserContext);
+  const [menuVisible, setMenuVisible] = useState(false);
+  const [activeSection, setActiveSection] = useState<string | null>(null);
+
+  const { dispatch } = useQueryParamsStore();
+
+  const navigate = useNavigate();
+
+  const handleMouseEnter = (sectionId: string) => {
+    setMenuVisible(true);
+    setActiveSection(sectionId);
```

Comment:
I think we need something here to only set the active section if there is content for it. Currently, if you hover over e.g. `Submit data`, and then move mouse sideways to `Overview`, the `Submit data` megamenu section still shows.

Reply:
Thanks @SandyRogers . I just pushed a fix to hide the megamenu content, for links that have no content section.

With regards to the external link icons. I can see what you're saying, however I think it's important to indicate when links open in a new tab. I just used that Icon cuz I saw it was already being used elsewhere in the app. 
But perhaps in the future we can choose a more subtle icon.